smdebug/core/access_layer/file.py

```diff
@@ -4,12 +4,10 @@
 
 # First Party
 from smdebug.core.logger import get_logger
-from smdebug.core.sagemaker_utils import is_sagemaker_job
 
 # Local
 from .base import TSAccessBase
 
-NON_SAGEMAKER_TEMP_PATH_PREFIX = "/tmp"
 SAGEMAKER_TEMP_PATH_SUFFIX = ".tmp"
```

Comment:
rename this variable too?

Reply:
renamed `SAGEMAKER_TEMP_PATH_SUFFIX` to `SMDEBUG_TEMP_PATH_SUFFIX`